corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -418,6 +420,9 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
         templateContext: function () {
             var paginateItems = paginateOptions(this.options.currentPage, this.options.pageCount);
             var casesPerPage = parseInt($.cookie("cases-per-page-limit")) || 10;
+            if (this.options.currentPage === 0) {
+                kissmetrics.track.event("Accessibility Tracking - Pagination First Page Loaded");
```

Comment:
The idea here was to log when pagination is present, but not necessarily interacted with. Initially I had tried to grab the button `#pagination-go-button` on page load, but it kept registering as being present. I was guessing it was simply hidden, but haven't yet been able to get that to work. Very open to suggestions if the current approach can be improved upon.

Reply:
What are you using to detect it? I'm looking at [this case list](https://staging.commcarehq.org/a/bosco/cloudcare/apps/v2/#%7B%22appId%22%3A%2204d9c23d5b064922916a12485bffc059%22%2C%22selections%22%3A%5B%221%22%5D%2C%22page%22%3Anull%2C%22search%22%3Anull%2C%22queryData%22%3A%7B%7D%2C%22sortIndex%22%3Anull%7D) and toggling between 10 & 25 items per page to make the pagination appear/disappear, and `$("#pagination-go-button").length` is returning either 1 or 0 depending on whether or not pagination is present.

Reply:
...thinking about this, it also matters where your code was. If it was running before the view was rendered, you might have been picking up the button from the view's template (the script tag).

Reply:
I was doing something like this at the top:
```
$(document ).ready( function ()  {
    var goButton = $("#pagination-go-button");
    if(goButton) {
      kissmetrics.track.event("Accessibility Tracking - Pagination Page Loaded");
    }
});
```
but should have been using `goButton.length` to check for the button.